### PR TITLE
Add new parameter for invert_threshold and change default value from …

### PR DESCRIPTION
**Pull request by stweil**
…0.5 to 0.7

Signed-off-by: Stefan Weil <sw@weilnetz.de>

### CODE REVIEW

File: src/ccmain/tesseractclass.cpp [hunk 1]
```diff
@@ -63,8 +63,11 @@ Tesseract::Tesseract()
                   "Break input into lines and remap boxes if present", this->params())
     , BOOL_MEMBER(tessedit_dump_pageseg_images, false,
                   "Dump intermediate images made during page segmentation", this->params())
-    , BOOL_MEMBER(tessedit_do_invert, true, "Try inverting the image in `LSTMRecognizeWord`",
+    , BOOL_MEMBER(tessedit_do_invert, true, "Try inverted line image if necessary",
```
Comment (amitdo) [inline src/ccmain/tesseractclass.cpp:66]: What about adding to the parameter description: 

`This parameter is deprecated and will be removed in version 6.0. Use the 'invert_threshold' parameter instead.`

Also add:

`// DEPRECATED`

or
```
// TODO (Stefan)
// Remove 'do_invert' in 6.0.
```
comment above.


This parameter deprecation should be mentioned in the release notes.

Comment (amitdo) [reply src/ccmain/tesseractclass.cpp:66]: This PR was merged. Can you please push the suggested changes as a new commit?

Comment (stweil) [reply src/ccmain/tesseractclass.cpp:66]: Done, see commit b8b6c158a7f356971d7ffc76b6d688606e05690d.

Comment (amitdo) [reply src/ccmain/tesseractclass.cpp:66]: Thanks.

**Pull request merged**